Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -117,30 +118,37 @@ public void SelectBucket(string name)
             _status.IsDisplayed &= string.IsNullOrEmpty(BucketName);
         }
 
-        public void CreateBucket()
+        public Response CreateBucket()
         {
             if (!CanCreate)
             {
-                return;
+                var emptyBucketNameResponse = new Response()
+                {
+                    ErrorMessage = BucketNameEmptyErrorMessage
+                };
+                OnBucketCreationFailure(emptyBucketNameResponse);
+                return emptyBucketNameResponse;
             }
 
             GetBootstrapDataResponse bootstrapResponse = _bootstrapUtility.GetBootstrapData();
 
             if (!bootstrapResponse.Success)
             {
                 OnBucketCreationFailure(bootstrapResponse);
-                return;
+                return bootstrapResponse;
             }
 
             CreateBucketResponse createResponse = _coreApi.CreateBucket(bootstrapResponse.Profile, bootstrapResponse.Region, BucketName);
 
             if (createResponse.Success)
             {
                 OnBucketCreated(bootstrapResponse.Profile, bootstrapResponse.Region, BucketName);
+                return bootstrapResponse;
             }
             else
             {
                 OnBucketCreationFailure(createResponse);
+                return bootstrapResponse;
```

Comment:
Why are you returning the boot strap response when the create bucket response is more recent?